gem/styles/gem-springster/modules/polls/_m-polls-form.scss

```diff
@@ -14,17 +14,20 @@
 .polls__bubble-icon {
```

Comment:
🍰 happy to land this, I've made some changes to the way the bubble icon is working, I'll update my changes